src/main/webapp/themes/bootstrap/pipeline-fullscreen.css

```diff
@@ -0,0 +1,61 @@
+body {
+    background: #fff;
+    color:#000;
+    font-size: 14px;
+}
+
+div.pipeline-main {
+    font-family: kalinga;
```

Comment:
We cannot use this font unfortunately since it is not available on all systems. We should at least fallback to sans-serif, but we could probably reuse the font-family from the default theme or what do you think?